app/views/dashboard/protocol_merges/perform_protocol_merge.js.coffee

```diff
@@ -0,0 +1 @@
+$("#flashes_container").html("<%= j render 'shared/flash' %>")
```

Comment:
Move these lines from `protocol_merges.js.coffee` to here

```
$('#master_protocol').val('')
$('#sub_protocol').val('')
```

Reply:
I think I'll leave them where they are for now.